tyrian/src/main/scala/tyrian/cmds/FileReader.scala

```diff
@@ -16,103 +18,88 @@ import scala.scalajs.js.typedarray
   */
 object FileReader:
 
+  val readImageCast: Result[js.Any] => Result[String] =
```

Comment:
question: should these `*Cast` vals be private?

Reply:
I like this improvement of extracting them out in general, but yes I agree that they don't belong on the public interface. :+1:

Reply:
Have been rather busy recently but I'll make a quick commit for this later tonight (or try to)

Reply:
Thanks @sloshy, I've been busy too, it happens. :slightly_smiling_face: 

Don't worry about it. I'm going to go ahead and merge this and then I'll quickly make the change - your efforts are appreciated! :pray:

Reply:
Ahh sorry things have been a little hectic over here. You're the best!